src/form/input/checkbox/Checkbox.tsx

```diff
@@ -6,19 +6,23 @@ import React from "react";
 import classNames from "classnames";
 
 export interface CheckboxInputItem {
-  testid?: string;
   id: string;
   content: React.ReactNode;
   inputProps: {
     htmlFor: string;
     value: string;
     name: string;
   };
+  context?: any;
```

Comment:
We can give the type for context as a generic which defaults to `any` if not provided

Reply:
I think that can be useful in some cases 👍 Also, same for `Radio` component